### DISCUSSION TITLE
App Font Size control

### DISCUSSION THREAD

**Discussion by kawak1320** [cstayyab/WALC#192] [Feature Requests] [answered]
Hi! Please add feature to be able to increase or decrease font size throughout the app,
Thank you very much!

**Comment by albakhtari** [comment, marked as answer]
You can zoom in and zoom out using `Ctrl`+`+` and `Ctrl`+`-`

**Reply by kawak1320** [reply]
thank you! i tried that but with num keypad and ctrl + mousewheel and it didn't work! with ansi keyboard only ctrl + - and ctrl+0 (my + is right side of -, and wont zoom in... for now ctrl 0 does the job, just reporting my use case! 
love the app!

**Reply by albakhtari** [reply]
Try `Ctrl` + `Shift` + `=`

**Reply by kawak1320** [reply]
well sir, idk why i haven't thought of that... LMAO
thank you very much!
